src/components/Code/example.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { trim } from 'lodash';
+import styled from 'styled-components';
+
+import { Example, Info } from '../../utils/example';
+import { Input } from '..';
+import Code from '.';
+
+const Highlight = styled.span`
+  color: ${props => props.theme.colors.accent.blue};
+`;
+
+export default class CodeExample extends React.Component {
+  state = {
+    value: '<ACCESS_KEY>',
+  };
+
+  onCopy = () => {
+    console.log('Copied');
+  };
+
+  onChange = e => {
+    this.setState({
+      value: e.target.value,
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <Input placeholder="Dynamic content" onChange={this.onChange} />
+
+        <Info>Long single line command</Info>
+        <Example>
+          <Code onCopy={this.onCopy}>
+            kubectl create secret generic -n weave cloudwatch
+            --from-literal=access-key-id=xyz
+            --from-literal=secret-access-key=abc
+          </Code>
+        </Example>
+
+        <Info>Multiline file</Info>
+        <Example>
+          <Code isFile onCopy={this.onCopy}>
+            {trim(`
```

Comment:
Perhaps it would make sense to wrap `children` in `trim` internally if the string is passed as the only param, i.e. `isString(children)`.

Reply:
Initially I had this but might there be an occasion where you want space before/after?

Reply:
I don't think so - if we currently don't have such a use case anywhere in the Service UI, I think you can assume it's not likely to happen :)